VimeoNetworking/VimeoNetworking/ResponseCache.swift

```diff
@@ -8,6 +8,19 @@
 
 import Foundation
 
+private typealias ResponseDictionaryClosure = (responseDictionary: VimeoClient.ResponseDictionary?) -> Void
+
+private protocol Cache
```

Comment:
I kinda think that if the cache work was outside then by doing it you kinda are obligated to make it maximally nice 🤓